astrocut/make_cube.py

```diff
@@ -85,7 +85,10 @@ def _configure_cube(self, file_list, **extra_keywords):
 
         # Working out the block size and number of blocks needed for writing the cube
         # without using too much memory
-        slice_size = image_shape[1] * len(self.file_list) * 2 * 4  # in bytes (float32)
+        try:
+            slice_size = image_shape[1] * len(self.file_list) * 2 * 4  # in bytes (float32)
+        except IndexError:
+            raise ValueError("One or more TICA FFIs were input. Please use ``TicaCubeFactory`` to process TICA FFIs.")
```

Comment:
I wonder if we can be so certain this is the error? couldn't an indexerror be caused by inputting other types of files? Perhaps the error message could be more general, and suggest that maybe the input is of the wrong ffi type?

Reply:
It depends on the structure of the FITS file, and whether or not the input file is structured the same as SPOC (for `CubeFactory`) and TICA (for `TicaCubeFactory`). But yes, in theory, this could break due to a non-TESS file. I've updated the error message accordingly.